external/netfx/netfx.depproj

```diff
@@ -25,5 +25,11 @@
     <FileToExclude Include="System.EnterpriseServices.Thunk" />
     <FileToExclude Include="System.EnterpriseServices.Wrapper" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">
```

Comment:
I believe these lines I added are required anyway, or else the depproj won't resolve those assemblies, but what we can do is that in the shims.proj we can remove them from non-uap builds.